demos/atlas/create_roi_from_atlas.m

```diff
@@ -1,12 +1,18 @@
 % (C) Copyright 2021 CPP ROI developers
 
-opt.roi.atlas = 'wang';
-opt.roi.name = {'V1v', 'V1d'};
+clear all;
+clc;
+
+% options : 'wang', 'neuromorphometrics', 'anatomy_toobox', 'visfAtlas'
+opt.roi.atlas = 'visfAtlas';
+
+% to get the list of possible run `getLookUpTable(opt.roi.atlas)`
+opt.roi.name = {'pFus', 'mFus', 'CoS'};
 opt.roi.dir = fullfile(pwd, 'derivatives', 'cpp_roi', 'group');
 
 spm_mkdir(opt.roi.dir);
 
-hemi = {'L', 'H'};
+hemi = {'L', 'R'};
```

Comment:
nice one!

can you something here to make sure that `extractRoiFromAtlas` will complain if we give it something different from R or L

https://github.com/cpp-lln-lab/CPP_ROI/blob/7870fd6ce7e6d5a1aabf2f95126babb09bd58afe/src/atlas/extractRoiFromAtlas.m#L22

Reply:
done, LMK if it ok